src/index.ts

```diff
@@ -11,6 +13,10 @@ import models from './models';
 import Example from './models/Example';
 
 const config = {};
+// configure loggin
+if (process.env.SENTRY_DSN !== 'false') {
```

Comment:
I suggest placing this in `config/sentry.config` 
Any opinions on this? I would like to see all that even smells like configuration put into separate files :-)